app/columns.tsx

```diff
@@ -302,8 +306,10 @@ export const experimentColumns: ColumnDef<RecipeOrBranchInfo>[] = [
 
       // XXX see https://bugzilla.mozilla.org/show_bug.cgi?id=1890055 for
       // re-enabling infobar code.
```

Comment:
Ironically, this comment is obsolete because it refers to a different thing where infobars were entirely disabled.  Let's remove it now.